database/sqlite.go

```diff
@@ -0,0 +1,73 @@
+package database
+
+import (
+	"errors"
+
+	"gorm.io/driver/sqlite"
+	"gorm.io/gorm"
+	"gorm.io/gorm/logger"
+)
+
+type SqliteDatabase struct {
+	*gorm.DB
+}
+
+func NewSqliteDatabase(path string) *SqliteDatabase {
+	db, err := gorm.Open(sqlite.Open(path), &gorm.Config{
+		Logger: logger.Default.LogMode(logger.Silent),
+	})
+	if err != nil {
+		// TODO handle database error
+		panic(err)
+	}
+	db.AutoMigrate(&Entry{})
+	return &SqliteDatabase{db}
+}
+
+func (db *SqliteDatabase) CreateEntry(entry *Entry) error {
```

Comment:
I think `Create` (or `CreateOne`) and `CreateMany` could be a little more simple, `Entry` appended doesn't seem necessary (it would read `db.Create`).

I'm thinking more about the usage of this package. I want to keep in concise.

Reply:
what if we need to add more tables later on, plus it's very descriptive and leaves no room for confusion